articles/api/authentication/api-authz/_highly-regulated.md

```diff
@@ -0,0 +1,213 @@
+| `prompt` | Can be used to force a particular prompt to display, e.g. `prompt=consent` will always display the consent prompt.|
+| `organization` | ID of the organization to use when authenticating a user. When not provided, if your application is configured to Display Organization Prompt, the user will be able to enter the organization name when authenticating. |
+
+## Remarks
+To make a call to the PAR endpoint, you must:
+- Set the request content type as `application/x-www-form-urlencoded`
+- Use `strings` for all passed parameters
+- Include an additional parameter for application authentication in the request (e.g. `client_secret`, or `client_assertion` and `client_assertion_type` for JSON Web Token Client Authentication, or pass a `client-certificate` and `client-certificate-ca-verified` header when using Mutual TLS).
+
+## Redirect from PAR to Authorize
+
+<%= include('../../../_includes/_http-method', {
+  "http_badge": "badge-primary",
+  "http_method": "GET",
+  "path": "/authorize",
+  "link": "#redirect-from-par-to-authorize"
+}) %>
+
+```http
+GET https://{yourDomain}/authorize
+  request_uri={yourRequestUri}&
+  client_id={yourClientId}
+```
+
+After calling the `/oauth/par` endpoint, redirect the end user with a `GET` call to the /authorize endpoint.
+
+:::note
+The `/authorize` endpoint will respond based on the parameters passed to the `/oauth/par` endpoint. If you request a `response_type` of code, you should receive an authorization code to use at the `/oauth/token` endpoint.
+:::
+
+### Request Parameters
+| Parameter        | Description |
+|:-----------------|:------------|
+| `client_id` <br/><span class="label label-danger">Required</span> | The `client_id` of your application. |
+| `request_uri` <br/><span class="label label-danger">Required</span> | The `request_uri` value that was received from the `/oauth/par` endpoint. |
+ 
+### Exchange an Authorization Code for a Token
+
+<%= include('../../../_includes/_http-method', {
+  "http_badge": "badge-primary",
+  "http_method": "POST",
+  "path": "/oauth/token",
+  "link": "#exchange-an-authorization-code-for-a-token"
+}) %>
+
+```http
+POST https://{yourDomain}/oauth/par
+Content-Type: 'application/x-www-form-urlencoded'
+  grant_type=code|code id_token&
+  client_id={yourClientId}&
+  code=CODE&
+ redirect_uri={https://yourApp/callback}&
+ code_verifier=CODE_VERIFIER
+```
+
+```javascript
+curl --request POST \
+  --url 'https://{yourDomain}/oauth/par' \
+  --header 'content-type: application/x-www-form-urlencoded' \
+--data 'grant_type=authorization_code& client_id={yourClientId}& code=CODE&redirect_uri={https://yourApp/callback}&code_verifier=CODE_VERIFIER'
+```
+
+> RESPONSE SAMPLE:
+``` json
+/**
+The `/oauth/token` endpoint will respond with a JSON object containing an `id_token` property, and potentially also a `refresh_token` if one was requested.
```

Comment:
```suggestion
The `/oauth/token` endpoint will respond with a JSON object containing an `access_token` property, and potentially also a `refresh_token` if one was requested. If you specified the authorization_details parameter at the beginning of the flow, it will also be added to the response to indicate the details of what has been authorized.
```

Reply:
I don't know if this text should appear here in the json example or be included as part of the remarks

Reply:
This line was originally associated with the Response, so I put it as commented out. Feel free to update it on the Google doc and I can update.